new_course/1_programming_fundementals/input_and_output.md

```diff
@@ -1,23 +1,73 @@
 <!-- This lesson will cover how to output things to the screen in Ruby and how to get input from the user. -->
 
 ### Introduction
-A brief summary about what this lesson is about and why the topics or concepts it covers are important.
+When we execute a computer program in Ruby Programming Language, most of the times it is required to either **output** or **input** a piece of information to a computer screen.
+For example, we may need to print (output) the information regarding a value of a number, your first and/or last name, etc. to the terminal. In contrast, we may require from the user to provide (input) the necessary information from the user screen (via using the keyboard) regarding e.g. a number (age), a small text (family’s name) etc.
 
 ### Learning Outcomes
-*Look through these now and then use them to test yourself after doing the assignment*
+* How can I produce an *input* ?
+* How can I produce an *output* ?
+* What is the basic syntax and use of the commands: *print*, *puts*, *gets*
+* What is the difference in input commands (*puts* /*print*) suggested?
 
-what the student is expected to know or be able to do by the end of this lesson
+### Syntax of Basic Commands
+* In order to output the information we want we shall use the **print** command !	
+```ruby
+print "Learning to code is FUN!" #=> will output to our screen the text Learning to code is FUN!
```

Comment:
Can we change the example output from "will output to our screen the text Learning to code is FUN!" to "Learning to code is FUN!" 

It's convention to show the exact output after #=> in ruby examples. Can we do that in all the examples please?